src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ImageListActionList.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing;
+
+namespace System.Windows.Forms.Design;
+
+internal class ImageListActionList : DesignerActionList
+{
+    private readonly ImageListDesigner _designer;
+
+    public ImageListActionList(ImageListDesigner designer)
+        : base(designer.Component)
+    {
+        _designer = designer;
+    }
+
+    public void ChooseImages() => EditorServiceContext.EditValue(_designer, Component, "Images");
+
+    public ColorDepth? ColorDepth
```

Comment:
Looks like making it nullable adds that blank row by treating null as a valid value, so I think we need to make it non-nullable.
```suggestion
    public ColorDepth ColorDepth
```